Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs

```diff
@@ -40,7 +43,7 @@ public static class Excel
         {
             if (options.ThrowErrorOnFailure)
             {
-                throw new Exception("Error while converting Excel file to JSON", ex);
+                throw new InvalidOperationException("Error while converting Excel file to JSON", ex);
             }
 
```

Comment:
https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L61
When appending single char to a string, you should use single quotes instead of double quotes to indicate that you are adding one character. StringBuilder has an overloaded method for adding a char.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L72
This can be a one-line else statement.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L81
This can be a one-line if statement.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L84
Please use single quotes.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L91
Please use single quotes.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L100
This can be a one-line if statement.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L104
This can be a one-line if statement. Also single quotes.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L118
This can be a one-line if statement. Also single quotes.

https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L124
This can be a one-line if statement. Also single quotes.

Reply:
Re one-line statements - matter of preference imo, and moreover they can be prone to errors when someone wants to add another statement and forgets to add braces. Also, when your if is slightly more complex then oneline statement is can be hard to read. Same goes for if-else constructs where if has braces and else does not.